src/extractors/custom/news.mynavi.jp/index.js

```diff
@@ -0,0 +1,47 @@
+export const NewsMynaviJpExtractor = {
+  domain: 'news.mynavi.jp',
+
+  title: {
+    selectors: [['meta[name="og:title"]', 'value']],
+  },
+
+  author: {
+    selectors: [
+      'body > div.wrapper > div.container > div.body > main > div.article-author > a',
+    ],
+  },
+
+  date_published: {
+    selectors: [['meta[name="article:published_time"]', 'value']],
+  },
+
+  dek: {
+    selectors: [['meta[name="og:description"]', 'value']],
+  },
+
+  lead_image_url: {
+    selectors: [['meta[name="og:image"]', 'value']],
+  },
+
+  content: {
+    selectors: [
+      'body > div.wrapper > div.container > div.body > main > article > div',
```

Comment:
Same comment here as with the author selector.

Reply:
I fixed as below.
```
      'main article div',
```